### PR TITLE
Bug 1907295: Gherkin script for improve management for helm ODC 2994

### PR DESCRIPTION
**Pull request by sanketpathak**
Epic: https://issues.redhat.com/browse/ODC-2994
Story:https://issues.redhat.com/browse/ODC-4843

### CODE REVIEW

File: frontend/packages/dev-console/integration-tests/features/helm/helm_feature_flag_ODC-5024.feature [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+Scenario: Enable the disabled helm features in console
+    Given user has disabled helm features
+    And the default "redhat-helm-repo" Helm Chart Repositories CR is available
+    When user opens "redhat-helm-repo" CR 
+    And user goes to YAML tab
+    And user removes "disabled: true" flag under "spec"
+    And user clicks on Save
+    Then user is able to see the Helm tab in the navigation menu
+    And user can see Helm Chart card in Add page
+    And user can see Helm Charts filter in the Developer Catalog page
```
Comment (gajanan-more) [inline frontend/packages/dev-console/integration-tests/features/helm/helm_feature_flag_ODC-5024.feature:33]: Please update Indentation

**Review by gajanan-more — CHANGES_REQUESTED** [review]

File: frontend/packages/dev-console/integration-tests/features/addFlow/developer-Catalog-Details.feature [hunk 2]
```diff
@@ -55,3 +55,14 @@ Feature: Developer Catalog Page
...
         And user will see Filter by Keyword field
         And user will see A-Z, Z-A sort by dropdown
+
+
+    @smoke, @manual
+    Scenario: Helm Charts filter specific to helm chart type
+        Given user has added multiple helm charts repositories
+        And user is at Developer Catalog page
+        When user clicks on Helm Charts type
+        And user select redhat-helm-repo from the list of Chart Repositories
```
[the remark this answers is not in the document]
Comment (sanketpathak) [reply frontend/packages/dev-console/integration-tests/features/addFlow/developer-Catalog-Details.feature]: Updated

File: frontend/packages/dev-console/integration-tests/features/helm/helm_feature_flag_ODC-5024.feature (hunk 1 above)
Comment (sanketpathak) [reply frontend/packages/dev-console/integration-tests/features/helm/helm_feature_flag_ODC-5024.feature]: Updated

**Review by gajanan-more — APPROVED** [review]

**Review by rohitkrai03 — APPROVED** [review]
LGTM

**Pull request merged**